app/importers/booking_bug/appointments.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
As we're not applying these magic comments anywhere else in the codebase it doesn't make sense to include it just for this file?